O/oneTBB/build_tarballs.jl

```diff
@@ -33,26 +44,24 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+platforms = expand_cxxstring_abis(supported_platforms())
 
 # Disable platforms unlikely to work
 filter!(p -> arch(p) ∉ ("armv6l", "armv7l"), platforms)
 
-# Windows with MinGW at the moment doesn't work, but it may change in the near
-# future, watch out <https://github.com/oneapi-src/oneTBB/pull/351>. See also
-# <https://stackoverflow.com/q/67572880/2442087>.
-filter!(!Sys.iswindows, platforms)
+#i686 mingw fails with errors about _control87
+filter!(p -> !Sys.iswindows(p) && arch(p) != "i686", platforms)
```

Comment:
Should have been
```julia
!Sys.iswindows(p) || arch(p) != "i686"
```
or
```julia
!(Sys.iswindows(p) && arch(p) == "i686")
```
Logic is hard

Reply:
ugh, patch PR incoming

Reply:
wait, wasn't `x86_64-mingw` building at one point? Wouldn't that first version drop that as well?